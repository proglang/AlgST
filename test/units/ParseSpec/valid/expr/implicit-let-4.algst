let ? : Int = 42 in ()
