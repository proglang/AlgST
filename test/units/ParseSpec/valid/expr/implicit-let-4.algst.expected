Exp.ILet ?
|
+- Type.Con Int
|
+- Exp.Lit
|  |
|  `- Lit.Int 42
|
`- Exp.Lit
   |
   `- Lit.Unit

